cmd/gateway-agent/main.go

```diff
@@ -44,6 +43,7 @@ func init() {
 	flag.Parse()
 	cfg.WireGuardConfigPath = path.Join(cfg.ConfigDir, "wg0.conf")
 	cfg.PrivateKeyPath = path.Join(cfg.ConfigDir, "private.key")
+	cfg.APIServerPasswordPath = path.Join(cfg.ConfigDir, "api_secret")
```

Comment:
suggestion for more consistent naming: `apiserver_password`

Reply:
Good point, will incorporate